lib/ui/dart_ui.cc

```diff
@@ -34,70 +34,278 @@
+  V(ImageDescriptor::initEncoded, 3)                                  \
+  V(ImmutableBuffer::init, 3)                                         \
+  V(ImmutableBuffer::initFromAsset, 3)                                \
+  V(ImageDescriptor::initRaw, 6)                                      \
+  V(IsolateNameServerNatives::LookupPortByName, 1)                    \
+  V(IsolateNameServerNatives::RegisterPortWithName, 2)                \
+  V(IsolateNameServerNatives::RemovePortNameMapping, 1)               \
+  V(NativeStringAttribute::initLocaleStringAttribute, 4)              \
+  V(NativeStringAttribute::initSpellOutStringAttribute, 3)            \
+  V(PlatformConfigurationNativeApi::DefaultRouteName, 0)              \
+  V(PlatformConfigurationNativeApi::ScheduleFrame, 0)                 \
+  V(PlatformConfigurationNativeApi::Render, 1)                        \
+  V(PlatformConfigurationNativeApi::UpdateSemantics, 1)               \
+  V(PlatformConfigurationNativeApi::SetNeedsReportTimings, 1)         \
+  V(PlatformConfigurationNativeApi::SetIsolateDebugName, 1)           \
+  V(PlatformConfigurationNativeApi::GetPersistentIsolateData, 0)      \
+  V(PlatformConfigurationNativeApi::ComputePlatformResolvedLocale, 1) \
+  V(PlatformConfigurationNativeApi::SendPlatformMessage, 3)           \
+  V(PlatformConfigurationNativeApi::RespondToPlatformMessage, 2)      \
+  V(DartRuntimeHooks::Logger_PrintDebugString, 1)                     \
+  V(DartRuntimeHooks::Logger_PrintString, 1)                          \
+  V(DartRuntimeHooks::ScheduleMicrotask, 1)                           \
+  V(DartRuntimeHooks::GetCallbackHandle, 1)                           \
+  V(DartRuntimeHooks::GetCallbackFromHandle, 1)                       \
+  V(DartPluginRegistrant_EnsureInitialized, 0)                        \
+  V(Vertices::init, 6)
 
-const uint8_t* GetSymbol(Dart_NativeFunction native_function) {
-  return g_natives->GetSymbol(native_function);
-}
+// List of native instance methods used as @FfiNative functions.
+// Items are tuples of ('class_name', 'method_name', 'parameter_count'), where:
+//   'class_name' is the name of the class containing the method.
+//   'method_name' is the name of the method.
+//   'parameter_count' is the number of parameters the method has including the
+//                     implicit `this` parameter.
+//
+// These are used to:
+// - Instantiate FfiDispatcher templates to automatically create FFI Native
+//   bindings.
+//   If the name does not match a native function, the template will fail to
+//   instatiate, resulting in a compile time error.
+// - Resolve the native function pointer associated with an @FfiNative function.
+//   If there is a mismatch between names or parameter count an @FfiNative is
+//   trying to resolve, an exception will be thrown.
+#define FFI_METHOD_LIST(V)                             \
+  V(Image, dispose, 1)                                 \
```

Comment:
Same comment about at least alphabetizing, and considering splitting back into separate files if possible.

Reply:
I moved `Image` to the right section, everything else looked alphabetized.

I don't have a strong preference either way for whether we move these to separate files, but I'm somewhat inclined to get this landed first before it ends up with merge conflicts.